_rules/element-lang-valid-de46e4.md

```diff
@@ -102,37 +107,57 @@ The `lang` attribute value is not a primary language subtag.
 ```html
 <html>
 	<body>
-		<article lang="dutch"></article>
+		<article lang="dutch">
+			Zij liepen een vreemde Tiki bar binnen, aan de rand van een dorpje aan het strand.
+		</article>
 	</body>
 </html>
 ```
 
 #### Failed Example 2
 
-The `lang` attribute value is not empty ("") and is not a primary language subtag.
+The `lang` attribute value is not empty (`""`) and is not a primary language subtag.
 
 ```html
 <html>
 	<body>
-		<article lang="#!"></article>
+		<article lang="#!">
+			They wandered into a strange Tiki bar on the edge of the small beach town.
+		</article>
 	</body>
 </html>
 ```
 
 #### Failed Example 3
 
-The `lang` attribute value consists of only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace) and is not a [valid language tag][].
+The `lang` attribute value consists of only [whitespace][] and is not a [valid language tag][].
 
 ```html
 <html>
 	<body>
-		<article lang=" ">
-			The quick brown fox jumped over the lazy dog
+		<article lang="  ">
+			They wandered into a strange Tiki bar on the edge of the small beach town.
 		</article>
 	</body>
 </html>
 ```
 
+#### Failed Example 4
+
+The `lang` attribute value has a valid language tag, but does not have a [visible][] [text node][] as a [descendant][] in the [flat tree][].
```

Comment:
Wouldn't that make this inapplicable?

Reply:
With the additional applicability - yes, and I think should continue to be included as an inapplicable example.